src/client/scripts/room.js

```diff
@@ -119,10 +115,45 @@ Template.room.events({
         }
 
         Cards.update({ _id: this._id}, { $inc: {likes: 1} });
+    },
+
+    "keydown #editor": function(keyPressed){
+        var pressed = keyPressed.which || keyPressed.keyCode;
+
+        if(keyPressed.ctrlKey && pressed == 83){
```

Comment:
okay i'll try that, i looked somewhere and it was said different browsers offer different values. like firefox returns undefined for .keycode